sidebars.js

```diff
@@ -11,21 +11,37 @@
 
 // @ts-check
 
-/** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
-const sidebars = {
-  // By default, Docusaurus generates a sidebar from the docs folder structure
-  tutorialSidebar: [{ type: 'autogenerated', dirName: '.' }],
-
-  // But you can create a sidebar manually
-  /*
-  tutorialSidebar: [
-    {
-      type: 'category',
-      label: 'Tutorial',
-      items: ['hello'],
-    },
-  ],
-   */
+const swap = (items, itemsWithHref) => {
+  const result = items.map((item) => {
+    if (item.type === 'category') {
+      return { ...item, items: swap(item.items, itemsWithHref) }
+    }
+
+    if (itemsWithHref[item.id]) {
+      item = {
```

Comment:
Any reason not to put the return here?

Reply:
Not really. I have to `return item` at the end either way, and I'm not a massive fan of returning things mid way through a function (either early return immediately or at the the end). I'm also not a fan of mutating a field like this but you can't win em all.